foursight_core/react/api/envs.py

```diff
@@ -0,0 +1,82 @@
+import copy
+from typing import Optional
+import os
+from dcicutils import ff_utils
+
+
+class Envs:
+
+    def __init__(self, known_envs: list):
```

Comment:
I'm nervous that you're just trusting the caller to know intimate details of environments rather than using the `env_utils` functions. This looks like new code, though may be it's just moved from elsewhere. The comment describes the arg as being the list returned by `app_utils.get_unique_annotated_environment_names`, but wouldn't it just want to call that function to avoid imposters being passed in? And once it's doing that, doesn't it just want the env_utils functions?

Reply:
Passing in really only for testing purposes. I could default it to None and for the default, non-test case, could have it call that get_unique_annotated_environment_names function, but that lives above in app_utils and would rather not have this (at least would-be) utility class have to reach back out to that.